File: jeorg-spring/jeorg-spring-flash/jeorg-spring-flash-set-1/jeorg-spring-flash-19/src/test/java/org/jesperancinha/sftd/flash19/transactional/cucumber/SpringFlash19CucumberLauncher.java
package org.jesperancinha.sftd.flash19.transactional.cucumber;


import io.cucumber.junit.CucumberOptions;
import org.jesperancinha.sftd.flash19.transactional.configuration.HikariConfiguration;
import org.jesperancinha.sftd.flash19.transactional.containers.AbstractTestContainersIT;
import org.junit.platform.suite.api.IncludeEngines;
import org.junit.platform.suite.api.Suite;
import org.springframework.context.annotation.Import;
import org.springframework.test.context.ContextConfiguration;

@Suite
@IncludeEngines("cucumber")
@CucumberOptions(features = {"classpath:/features/albums.save.feature"})
@Import(HikariConfiguration.class)
@ContextConfiguration(initializers = AbstractTestContainersIT.DockerPostgresDataInitializer.class)
public class SpringFlash19CucumberLauncher {


}

File: jeorg-spring/jeorg-spring-flash/jeorg-spring-flash-set-1/jeorg-spring-flash-19/src/test/kotlin/org/jesperancinha/sftd/flash19/transactional/configuration/cucumber/SpringFlash19CucumberLauncher.kt
package org.jesperancinha.sftd.flash19.transactional.configuration.cucumber

import io.cucumber.junit.CucumberOptions
import org.jesperancinha.sftd.flash19.transactional.configuration.HikariConfiguration
import org.jesperancinha.sftd.flash19.transactional.configuration.containers.AbstractTestContainersKotlinIT
import org.junit.platform.suite.api.IncludeEngines
import org.junit.platform.suite.api.Suite
import org.springframework.context.annotation.Import
import org.springframework.test.context.ContextConfiguration

@Suite
@IncludeEngines("cucumber")
@CucumberOptions(features = ["classpath:/features/albums.save.feature"])
@Import(
    HikariConfiguration::class
)
@ContextConfiguration(initializers = [AbstractTestContainersKotlinIT.DockerPostgresDataInitializer::class])
class SpringFlash19CucumberLauncher 